package-dev/Plugins/macOS/SentryNativeBridge.m

```diff
@@ -66,7 +68,17 @@ void SentryNativeBridgeOptionsSetInt(const void *options, const char *name, int3
 void SentryNativeBridgeStartWithOptions(const void *options)
 {
     NSMutableDictionary *dictOptions = (__bridge_transfer NSMutableDictionary *)options;
-    [SentrySDK performSelector:@selector(startWithOptions:) withObject:dictOptions];
+    id sentryOptions = [[SentryOptions alloc]
+        performSelector:@selector(initWithDict:didFailWithError:)
+        withObject:dictOptions withObject:nil];
+
+    if (sentryOptions == nil)
```

Comment:
m: Without and error argument in the line 73, sentryOptions will never be nil

Reply:
Thanks for pointing that out!